pegasus/sites.v3/code.org/public/js/font_awesome_icon_class.js

```diff
@@ -0,0 +1,4 @@
+// Adds a .fa-fw class to all icon elements
+// Used to maintain consistent spacing when
+// icons have varying widths and heights
+document.querySelectorAll("i").forEach((icon) => icon.classList.add("fa-fw"));
```

Comment:
Not sure if we really need this script. I guess there's a minor possibility that we'll need an icon of custom size I don't see an easy workaround for that case in this simple,entation.
I'm open to discussion, but in my opinion it's better to set fa-fw directly to every icon we want to add.

Also I think it's better to move icon related changes to separate PR so it'll be easier to find this PR later in the git log.

Reply:
* By ` icon related changes` I mean changes that adds `fa-fw `and sets` line-height: 1.25em;`

Reply:
I think in general it makes more sense to automatically add it to every icon so they're all consistent. This may not be the correct solution for Studio, but I think it is for Pegasus considering how they're used. Happy to consult w/ others though to get their opinions.

----

> there's a minor possibility that we'll need an icon of custom size and I don't see an easy workaround for that case in this implementation.

We can add in-line styles on a particular icon if it needs to be overridden — I think that's a more sustainable solution for a less common use case, than needing to remember to add the `fa-fw` class to every icon individually. We have a lot of them on the Pegasus rebrand pages! 😄  

> Also I think it's better to move icon related changes to separate PR so it'll be easier to find this PR later in the git log.

And then fix the styles in a separate one too? Should I hold off until we decide on the question above?

Reply:
If all the changes in this PR are caused by updating icons sizing, I guess it's ok to keep them in this PR.

Reply:
This approach sounds reasonable to me. Thanks for the discussion here!